src/sentry/api/endpoints/organization_config_repositories.py

```diff
@@ -26,9 +31,9 @@ def get(self, request, organization):
                     }
                 )
 
-        if features.has('organizations:internal-catchall', organization, actor=request.user):
-            for provider_id in integrations_provider_bindings:
-                provider = integrations_provider_bindings.get(provider_id)(id=provider_id)
+        for provider_id in integrations_provider_bindings:
+            provider = integrations_provider_bindings.get(provider_id)(id=provider_id)
+            if provider_id in early_adopters:
```

Comment:
and then here you could just do:
```python
has_catchall = features.has('organizations:internal-catchall', organization, actor=request.user)
for provider_id in integrations_provider_bindings:
    if has_catchall or (has_github_apps and provider_id == 'integrations:github'):
        provider = integrations_provider_bindings.get(provider_id)(id=provider_id)
        providers.append(...
```

if you don't do something like that, won't you temporarily disable bitbucket for us?